src/FakerDotNet/Data/SlackEmojiData.cs

```diff
@@ -0,0 +1,920 @@
+            ":x:",
+            ":o:",
+            ":100:",
+            ":end:",
+            ":back:",
+            ":on:",
+            ":top:",
+            ":soon:",
+            ":cyclone:",
+            ":m:",
+            ":ophiuchus:",
+            ":six_pointed_star:",
+            ":beginner:",
+            ":trident:",
+            ":warning:",
+            ":hotsprings:",
+            ":recycle:",
+            ":anger:",
+            ":diamond_shape_with_a_dot_inside:",
+            ":spades:",
+            ":clubs:",
+            ":hearts:",
+            ":diamonds:",
+            ":ballot_box_with_check:",
+            ":white_circle:",
+            ":black_circle:",
+            ":radio_button:",
+            ":red_circle:",
+            ":large_blue_circle:",
+            ":small_red_triangle:",
+            ":small_red_triangle_down:",
+            ":small_orange_diamond:",
+            ":small_blue_diamond:",
+            ":large_orange_diamond:",
+            ":large_blue_diamond:",
+            ":black_small_square:",
+            ":white_small_square:",
+            ":black_large_square:",
+            ":white_large_square:",
+            ":black_medium_square:",
+            ":white_medium_square:",
+            ":black_medium_small_square:",
+            ":white_medium_small_square:",
+            ":black_square_button:",
+            ":white_square_button:",
+            ":clock1:",
+            ":clock2:",
+            ":clock3:",
+            ":clock4:",
+            ":clock5:",
+            ":clock6:",
+            ":clock7:",
+            ":clock8:",
+            ":clock9:",
+            ":clock10:",
+            ":clock11:",
+            ":clock12:",
+            ":clock130:",
+            ":clock230:",
+            ":clock330:",
+            ":clock430:",
+            ":clock530:",
+            ":clock630:",
+            ":clock730:",
+            ":clock830:",
+            ":clock930:",
+            ":clock1030:",
+            ":clock1130:",
+            ":clock1230:"
+        };
+
+        public static readonly IEnumerable<string> Custom = new[]
+        {
+            ":beryl:",
+            ":bowtie:",
+            ":crab:",
+            ":cubimal_chick:",
+            ":dusty_stick:",
+            ":feelsgood:",
+            ":finnadie:",
+            ":fu:",
+            ":goberserk:",
+            ":godmode:",
+            ":hurtrealbad:",
+            ":metal:",
+            ":neckbeard:",
+            ":octocat:",
+            ":piggy:",
+            ":pride:",
+            ":rage1:",
+            ":rage2:",
+            ":rage3:",
+            ":rage4:",
+            ":rube:",
+            ":simple_smile:",
+            ":slack:",
+            ":squirrel:",
+            ":suspect:",
+            ":taco:",
+            ":trollface:"
+        };
+
+        public static readonly IEnumerable<string> Emoji = new[]
+        {
+            "{people}",
+            "{nature}",
+            "{food_and_drink}",
```

Comment:
These need to be updated to reflect the name of the method. Same with the ones below 👇